docs/quickstart/docker.md

```diff
@@ -15,54 +15,31 @@ Coder with Docker has the following advantages:
 
 1.  [Install and launch Coder](../install)
 
-    You will specify `CODER_ACCESS_URL=http://localhost:7080` since we're using
+    Specify `CODER_ACCESS_URL=http://localhost:7080` since we're using
```

Comment:
"Next, export variables that will be used for the coder process." 

jumping in like this at the beginning of an instruction is confusing to the user - better to be verbose + explicit.